docs/services/roundcube.md

```diff
@@ -0,0 +1,55 @@
+# Roundcube
+
+[Roundcube](https://roundcube.net/) is a browser-based multilingual IMAP client with an application-like user interface. It provides full functionality you expect from an email client, including MIME support, address book, folder manipulation, message searching and spell checking.
+
+## Dependencies
+
+This service requires the following other services:
+
+- a [Postgres](postgres.md) database
+- a [Traefik](traefik.md) reverse-proxy server
+
+## Configuration
+
+To enable this service, add the following configuration to your `vars.yml` file and re-run the [installation](../installing.md) process:
+
+```yaml
+########################################################################
+#                                                                      #
+# roundcube                                                            #
+#                                                                      #
+########################################################################
+
+roundcube_enabled: true
+
+roundcube_hostname: mash.example.com
+
+roundcube_path_prefix: "roundcube"
```

Comment:
Ah, sorry, that was a typo.  I fixed it now.

Nevertheless, I haven't been able to test whether the path prefix works.  I will try to do it today, but I'd appreciate if you could also take a look, just in case.

Thanks.!

Reply:
I was able to perform a more comprehensive test and found a problem with the traefik labels, which I've now fixed.  The role now works fine with and without a path prefix.